edi_835_parser/loops/claim.py

```diff
@@ -31,26 +31,26 @@ class Claim:
 	]
 
 	def __init__(
-		self,
-		claim: ClaimSegment = None,
-		entities: List[EntitySegment] = None,
-		services: List[ServiceLoop] = None,
-		references: List[ReferenceSegment] = None,
-		dates: List[DateSegment] = None,
-		amount: AmountSegment = None,
-		inpatient: InpatientAdjudicationSegment = None,
-		outpatient: OutpatientAdjudicationSegment = None,
-		adjustment: ClaimAdjustmentSegment = None,
+			self,
+			claim: ClaimSegment = None,
+			entities: List[EntitySegment] = None,
+			services: List[ServiceLoop] = None,
+			references: List[ReferenceSegment] = None,
+			dates: List[DateSegment] = None,
+			amounts: List[AmountSegment] = None,
+			inpatient: InpatientAdjudicationSegment = None,
+			outpatient: OutpatientAdjudicationSegment = None,
+			adjustments: List[ClaimAdjustmentSegment] = None
```

Comment:
heck yea

Reply:
Tests pass.   I am going to set up circle ci.